examples/blended_pairwise_with_sparsify.jl

```diff
@@ -116,6 +111,27 @@ x0 = deepcopy(x00)
     lp_solver=lp_solver,
 );
 
+trajectoryBPCG_as_sparse = []
+callback = build_callback(trajectoryBPCG_as_sparse)
+
+active_set_sparse = FrankWolfe.ActiveSetSparsifier(FrankWolfe.ActiveSet([1.0], [x00], similar(x00)), HiGHS.Optimizer())
+
+@time x, v, primal, dual_gap, _ = FrankWolfe.blended_pairwise_conditional_gradient(
+    f,
+    grad!,
+    lmo,
+    copy(x00),
+    max_iteration=k,
+    line_search=FrankWolfe.Shortstep(2.0),
+    print_iter=k / 10,
+    memory_mode=FrankWolfe.InplaceEmphasis(),
+    verbose=true,
+    trajectory=true,
+    callback=callback,
+    sparsify=false,
+#    squadratic=true, # activate to see the effect of the numerical precision of the LP solver
```

Comment:
these two params/lines should be probably removed now that we use the active set with it.

Reply:
yes I was still planning to clean up the parameters after adding some tests